sdk/digitaltwins/digitaltwins/rollup.base.config.js

```diff
@@ -0,0 +1,153 @@
+import nodeResolve from "@rollup/plugin-node-resolve";
+import multiEntry from "@rollup/plugin-multi-entry";
+import cjs from "@rollup/plugin-commonjs";
+import replace from "@rollup/plugin-replace";
+import { terser } from "rollup-plugin-terser";
+import sourcemaps from "rollup-plugin-sourcemaps";
+import shim from "rollup-plugin-shim";
+import json from "@rollup/plugin-json";
+import * as path from "path";
+import inject from "@rollup/plugin-inject";
+
+const pkg = require("./package.json");
+const depNames = Object.keys(pkg.dependencies);
+const devDepNames = Object.keys(pkg.devDependencies);
+const input = "dist-esm/src/index.js";
+const production = process.env.NODE_ENV === "production";
+
+const ignoreKnownWarnings = (warning) => {
+  if (warning.code === "THIS_IS_UNDEFINED") {
+    // This error happens frequently due to TypeScript emitting `this` at the
```

Comment:
this sounds weird - why would TS stick `this` into module scope?

Reply:
Good catch. Removed.

---
In reply to: [470199500](https://github.com/Azure/azure-sdk-for-js/pull/10560#discussion_r470199500) [](ancestors = 470199500)